### ISSUE TITLE
On sonar, the master branch is not udpated!

### ISSUE THREAD

**Issue by JulienPeloton** [astrolabsoftware/fink-broker#251]
**Describe the issue**
In travis, sonar analysis is triggered only if `$TRAVIS_PULL_REQUEST_SLUG == "astrolabsoftware/fink-broker"`. This means PRs originated from the main repo (no fork) - but only PR builds! Therefore, push builds from the master were not taken into account (especially merged PR...). This has no incidence on the way we operated (sonar analysis was right inside PR), but information was never pushed back to master...

action item: replace `TRAVIS_PULL_REQUEST_SLUG` by `TRAVIS_REPO_SLUG`. This way, only builds from `astrolabsoftware/fink-broker` will be enabled (no forks), but both pr and push builds will be taken into account.

**Comment by JulienPeloton** [comment]
Actually #252 does not solve the problem as `TRAVIS_REPO_SLUG` is always defined as `owner/name` (this variable concerns only the repository currently being built). A better hack would be to use:

```shell
# The first activates for build, while the second one activates for PR originating from the upstream only.
if [ $TRAVIS_PULL_REQUEST == false ] || [ $TRAVIS_PULL_REQUEST_SLUG == "astrolabsoftware/fink-broker"]; then
  ...
fi
```

**Comment by JulienPeloton** [comment]
Fixed in #253
